frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx

```diff
@@ -18,6 +19,13 @@ interface MissionDisplayProps {
     mission: Mission
 }
 
+interface RemoveMissionDialog {
+    confirmDeleteDialogOpen: boolean
+    mission: Mission
+    setConfirmDeleteDialogOpen: React.Dispatch<React.SetStateAction<boolean>>
+    onDeleteMission: (mission: Mission) => void
+}
```

Comment:
```suggestion
interface RemoveMissionDialogProps {
    confirmDeleteDialogOpen: boolean
    mission: Mission
    setConfirmDeleteDialogOpen: React.Dispatch<React.SetStateAction<boolean>>
    onDeleteMission: (mission: Mission) => void
}
```

![image](https://github.com/equinor/flotilla/assets/33518988/2096a4ec-0b46-4b93-861f-77e5472cff28)

Gets warning when reusing the same name for both the interface and the function. Should rename one of them